### ISSUE TITLE
I made a Form-Fillable/Auto-Calc Character Sheet

### ISSUE THREAD

**Issue by ZoeyProgrammer** [ElectricCoffee/SirenRPG#9]
I made this for my own Group, but wanted to share it so others could use it as well, if they wanted to.
It was made for the Beta 2.1.X and it's not perfect, but it should definitly do its job fair enough.

Hope it helps ^^
[Siren_CharacterSheet_AutoCalc.pdf](https://github.com/ElectricCoffee/SirenRPG/files/6448458/Siren_CharacterSheet_AutoCalc.pdf)

**Comment by ElectricCoffee** [comment]
Your own group? I'm glad people are playing this!

BTW, would you mind if we were to add this to the next release as a download? With credit of course! :)

**Comment by ZoeyProgrammer** [comment]
Yes, it seemed like a very interesting Universal System so we're using it for an upcoming campaign.
With little changes and homerules here and there to make it fit better for the setting, as customary for a universal system.

But sure, feel free to use it / edit it further as you like.
I'd be glad if my little contribution helped someone with the same interests ^^

**Comment by ElectricCoffee** [comment]
I tried making an auto-filling one a few times, but some of the boxes never updated correctly no matter how hard we tried.

May I ask, how did you come across the game? I don't often get much in the way of feedback or other interactions from non-testers, so it's interesting to see how people arrive here.

**Comment by ZoeyProgrammer** [comment]
Well, one of the people I play with found it randomly on twitter and told us about it. And since we were looking for an universal system and didn't particularily like the larger ones (GURPs, etc.) this one looked quite attractive, especially with how open-ended it is and rather simple to get into as well.

Then I saw that there was no Auto-Filling one, and since we primarily play online I quickly made one (pretty quick and dirty and not sure if it's compatable with other pdf readers other than Adobe Acrobat DC)

And that's what brings us here pretty much.

**Comment by ElectricCoffee** [comment]
Thanks a lot for showing interest in it, here's hoping it'll serve you well in your adventures to come! 

I'd love to know how it plays for you and what might be improved upon in the future :)
